build/yamls/antrea-coverage.yml

```diff
@@ -0,0 +1,1145 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
What's generating this YAML? How do we keep it up-to-date?

Please update `hack/generate-manifest.sh` so it can be auto-generated.

Reply:
ok, thanks Antoni, working on it.